.travis/stages/verify/check-links-and-yaml-syntax/check-links

```diff
@@ -15,28 +16,37 @@ fi
 
 function checkUri() {
   local -r uri="$1"
-  
-  local -r curlResult=$(curl -s --head "$uri" | head -n 1)
-  echo "$uri -- $curlResult"  
-  # Temporarily disabled. We're unable to connect to A&A.org from Travis.
-
-  # echo $curlResult |  grep -qE  "HTTP/2 200|(200|302) (OK|Found)" \
-  #  || reportBrokenUri "$uri"
+  local code
+  if ! code=$(curl -Lsf -o /dev/null -w "%{http_code}" "$uri"); then
+    reportBrokenUri "$uri" "$code"
+    FAILURE=1
+  else
+    echo "Successfully checked '$uri', returned $code"
+  fi
 }
 
 function reportBrokenUri() {
-  FAILURE=1 
   local -r red="\e[38;5;196m"
   local -r blue="\e[38;5;75m"
-  echo -en "Error: ${red}$uri${end} ${blue}"
-  curl -s --head "$uri" | head -n 1 | sed 's|HTTP/[12][.01]?||'
-  echo -en "${end}"
+  local -r error_reason=$([[ $2 -ne 0 ]] && echo "Failed with Code: $2" || echo "Connection Failure!")
+  echo -e "Error: ${red}${1}${end} ${blue}${error_reason}${end}"
+}
+
+function checkIsOnIgnoreList() {
+  for ignored_uri in "${IGNORE_LIST[@]}"; do
+    if [[ "$1" = "$ignored_uri" ]]
```

Comment:
Nit, the other scripts have a convention to assign parameters to variables in functions. Thus any parameter passed to a function receives a name, you see off-the-bat as well how many parameters are expected to be passed to a function. It also means that a variable like `$1` is only ever referenced once in a function and that is only to assign it to another variable that then gives it a name.

This was undocumented, I added a note about this convention here: https://github.com/triplea-game/triplea/wiki/Shell-Script-Conventions